packages/test/local-server-tests/package.json

```diff
@@ -109,6 +109,14 @@
 		"webpack": "^5.82.0",
 		"webpack-cli": "^4.9.2"
 	},
+	"fluidBuild": {
+		"tasks": {
+			"build:test": [
+				"^tsc",
```

Comment:
For my/reviewer reference, packages have `build:test` depend on `^tsc` + `^api-extractor:commonjs` when they are commonjs and on `^build:esnext` + `^api-extractor:esnext` when they're esm.

Tangent: we should probably make this package esm, but not in this PR.

Reply:
Yep. And we prefer to have `build:test:cjs|esm` for clarity. `build:test` can exist to run either or both of those but doesn't need dependencies itself. We just need to make sure there are no more `build:test` that do things directly.

Re Tangent: I knew about this. For now, I thought it was good to have something that runs CommonJS. There are some tasks to make sure we have correct coverage on ESM / CJS.